comment/api/serializers.py

```diff
@@ -22,10 +24,14 @@ def get_profile_model():
 def get_user_fields():
     user_model = get_user_model()
     fields = user_model._meta.get_fields()
+    api_fields = settings.COMMENT_USER_API_FIELDS
+    api_fields.append('profile')
```

Comment:
This just assumes the user has set it to a list. They may also use a tuple. You may probably change this to something like this to be more flexible:
```suggestion python
    api_fields = list(settings.COMMENT_USER_API_FIELDS)
    api_fields.append('profile')
```